templates/octaviaapi/config/octavia.conf

```diff
@@ -89,7 +89,7 @@ auth_type = password
 project_domain_name = Default
 user_domain_name = default
 region_name = regionOne
-username = nova
+username = {{ .ServiceUser }}
```

Comment:
Do we really need this user credential ? I guess octavia just uses the one in service_auth instead.

Reply:
I agree, service_auth should be enough